packages/block-editor/src/components/link-control/test/fixtures/index.js

```diff
@@ -7,29 +7,29 @@ export const fauxEntitySuggestions = [
 	{
 		id: uniqueId(),
 		title: 'Hello Page',
-		type: 'Page',
+		type: 'page',
```

Comment:
The correct type should be lowercased?

Reply:
Yes this should match what the API returns which is lowercase.